src/runtime/types.ts

```diff
@@ -24,6 +29,10 @@ export interface Authentication {
    * @default 'X-XSRF-TOKEN'
    */
   csrfHeader?: string
+  /**
+   * Token storage handlers to be used by the client.
+   */
+  tokenStorage?: TokenStorage
```

Comment:
I think this is supposed to be used as app options interface and not ModuleOptions, you can check how it is defined in `nuxt-auth-sanctum` as well